todo-frontend-angular/src/app/todo/SubtaskModel.tsx

```diff
@@ -0,0 +1,9 @@
+export type SubtaskModel = {
```

Comment:
So in Angular we don't use .tsx since that is JXS syntax :D 
The correct naming would be: subtask.model.ts and should be put into a model folder.

furthermore I think you should declare it as an Interface instead :)